js/styles/atlas.css

```diff
@@ -96,12 +96,11 @@ body {
 }
 
 .heading {
```

Comment:
removed the bottom border from header elements to reduce confusion from all the horizontal lines it was introducing in some interfaces